src/SDKs/Reservations/Microsoft.Azure.Management.Reservations/Microsoft.Azure.Management.Reservations.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Reservations</PackageId>
+    <Description>Microsoft.Azure.Management.Reservations Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.Reservations</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
```

Comment:
@corquiri sorry forgot to mention this earlier in my review, but you are missing packageReleaseNotes take a look at this example
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10
having package release notes helps your customers/users to know what is in the package (treat it as what you would like to tell your users what this nuget package has)

Reply:
Release note added